packages/components/MIGRATING.md

```diff
@@ -579,3 +582,70 @@ Changes:
     </Tooltip.Content>
 </Tooltip.Root>
 ```
+
+## Slider
+
+Changes:
+
+-   **Controlled and Uncontrolled States**:
+
+    -   `value` (controlled) and `defaultValue` (uncontrolled) props are now arrays reflecting support for range sliders in the new component.
+    -   The `onChange` event now returns an array of numbers reflecting the new multi-thumb capabilities.
+
+-   **Event Handling**:
+
+    -   The new `onCommit` event is introduced for actions when user interaction ends, akin to an `onBlur`.
+    -   Error handling via `onError` as this component can no longer be in an error state. Use controlled mode to limit and alert the user.
+
+
+-   **Removed Properties**:
+
+    -   `stepMultiplier` is no longer needed.
+    -   `showMinMax` is no longer supported.
+    -   `showMinMax` is no longer supported. 
+    -   Additional UI flourishes should be added outside the component as needed.
+
+        ```tsx
+        <Slider />
+        <span className="tw-mr-3">
+            {min}%-{max}%
+        </span>
+        ```
+
+#### Old
+```tsx
+// Old Slider
+<Slider
+    id="old-slider"
+    label="Volume"
+    value={50}
+    min={0}
+    max={100}
+    showMinMax={true}
+    step={5}
+    stepMultiplier={10}
+    valueSuffix="%"
+    disabled={false}
+    onError={(errorCode) => console.error("Error:", errorCode)}
+    onChange={(value) => console.log("Value:", value.raw, value.withSuffix)}
+    data-test-id="slider-test"
+    aria-label="Adjust volume level"
+/>
+```
+#### New
+```tsx
+<Slider
+    id={"new-slider"}
+    value={[50]} // use for controlled mode
+    defaultValue={[50]} // use for uncontrolled mode
+    min={0}
+    max={100}
+    step={5}
+    disabled={false}
+    onChange={(values) => console.log("Value:", values)}
+    onCommit={(values) => console.log("Final Value:", values)}
+    data-test-id="slider-test"
+    aria-label="Adjust volume level"
+    aria-describedby="volume-description"
```

Comment:
Could you make the example correct by using `<Label`?

Reply:
yep my bad